roqoqo-qryd/src/api_devices.rs

```diff
@@ -350,7 +357,11 @@ impl QrydEmuSquareDevice {
     /// * `f64` - The PhaseShiftedControlledPhase phase shift.
     ///
     pub fn phase_shift_controlled_phase(&self, theta: f64) -> Option<f64> {
-        phi_theta_relation(&self.controlled_phase_phase_relation, theta)
+        if let Ok(phase_shift_value) = f64::from_str(&self.controlled_phase_phase_relation) {
+            Some(phase_shift_value)
+        } else {
+            phi_theta_relation(&self.controlled_phase_phase_relation, theta)
+        }
```

Comment:
Here we do not want to have the option to overwrite the relation with a constant float value, because it should always depend on theta.